components/checkbox/index.css

```diff
@@ -471,7 +471,6 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Checkbox-box {
-
```

Comment:
I'm seeing some formatting changes in unrelated components like checkbox and search and tag. The branch likely just needs to be rebased with the latest `spectrum-two`?